<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Canciones</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lnColeccion"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBuscar">
            <property name="text">
             <string>Buscar</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/images/edit-find.png</normaloff>:/images/edit-find.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listColeccion">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragOnly</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnNuevo">
            <property name="text">
             <string>Nuevo</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/images/document-new.png</normaloff>:/images/document-new.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditar">
            <property name="text">
             <string>Editar</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAgregar">
            <property name="text">
             <string>Agregar</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Biblia</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Lista de reproducci√≥n</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="listLista">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="movement">
            <enum>QListView::Free</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="btnSubir">
             <property name="text">
              <string>Subir</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/arrow-up.png</normaloff>:/images/arrow-up.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnBajar">
             <property name="text">
              <string>Bajar</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/arrow-down.png</normaloff>:/images/arrow-down.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnQuitar">
             <property name="text">
              <string>Quitar</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Controles</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QListWidget" name="listControles">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btnInicio">
             <property name="text">
              <string>Inicio</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/go-first-view.png</normaloff>:/images/go-first-view.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAnterior">
             <property name="text">
              <string>Anterior</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/go-previous-view.png</normaloff>:/images/go-previous-view.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSiguiente">
             <property name="text">
              <string>Siguiente</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/go-next-view.png</normaloff>:/images/go-next-view.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnFin">
             <property name="text">
              <string>Fin</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/go-last-view.png</normaloff>:/images/go-last-view.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Vista previa</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QToolButton" name="btnFontType">
             <property name="text">
              <string>Tipo de letra</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/application-x-font-ttf.png</normaloff>:/images/application-x-font-ttf.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnFontColor">
             <property name="text">
              <string>Color de letra</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/format-text-color.png</normaloff>:/images/format-text-color.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnBGColor">
             <property name="text">
              <string>Fondo</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/fill-color.png</normaloff>:/images/fill-color.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnBGImage">
             <property name="text">
              <string>Imagen</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/insert-image.png</normaloff>:/images/insert-image.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGraphicsView" name="gvVistaPrevia">
             <property name="minimumSize">
              <size>
               <width>266</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>266</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnComenzar">
               <property name="text">
                <string>Comenzar</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/images/media-playback-start.png</normaloff>:/images/media-playback-start.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSoloImagen">
               <property name="text">
                <string>S√≥lo imagen</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/images/imagegallery.png</normaloff>:/images/imagegallery.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPantallaBlanca">
               <property name="text">
                <string>Pantalla negra</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPantallaNegra">
               <property name="text">
                <string>Pantalla blanca</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDetener">
               <property name="text">
                <string>Detener</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/images/media-playback-stop.png</normaloff>:/images/media-playback-stop.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuVer"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
